### PR TITLE
core: Configurable name length Fix #399

### LINKED ISSUE #399: Configurable name length
Motivation
----------

Make the length of the name configurable

### PR DESCRIPTION
**Pull request by vingar**

### CODE REVIEW

**Comment by vingar** [comment]
500 for name works on all tested backend (see previous test executions)

File: lib/rucio/common/schema/atlas.py [hunk 1]
```diff
@@ -43,17 +43,21 @@
                      "T0 Export", "T0 Tape", "Upload/Download (Job)",
                      "Upload/Download (User)", "User Subscriptions"]}
 
+SCOPE_LENGTH = 25
+
 SCOPE = {"description": "Scope name",
          "type": "string",
-         "pattern": "^[a-zA-Z'_'-.0-9]{1,30}$"}
+         "pattern": "^[a-zA-Z'_'-.0-9]{1,%s}$" % SCOPE_LENGTH}
```
Comment (cserf) [inline lib/rucio/common/schema/atlas.py:50]: Shouldn't we allow more characters ?

File: lib/rucio/common/schema/atlas.py [hunk 2]
```diff
@@ -43,17 +43,21 @@
...
 R_SCOPE = {"description": "Scope name",
            "type": "string",
            "pattern": "\\w"}
 
+NAME_LENGTH = 250
+
 NAME = {"description": "Data Identifier name",
         "type": "string",
-        "pattern": "^[A-Za-z0-9][A-Za-z0-9\\.\\-\\_]{1,255}$"}
+        "pattern": "^[A-Za-z0-9][A-Za-z0-9\\.\\-\\_]{1,%s}$" % NAME_LENGTH}
```
Comment (cserf) [inline lib/rucio/common/schema/atlas.py:60]: Same comment as above

Comment (vingar) [reply lib/rucio/common/schema/atlas.py:60]: Same comment as above

**Review by cserf — APPROVED** [review]

**Pull request merged**